pkgs/applications/networking/kubo/default.nix

```diff
@@ -33,9 +33,9 @@ buildGoModule rec {
 
   postPatch = ''
     substituteInPlace 'misc/systemd/ipfs.service' \
-      --replace '/usr/bin/ipfs' "$out/bin/ipfs"
+      --replace '/usr/local/bin/ipfs' "$out/bin/ipfs"
     substituteInPlace 'misc/systemd/ipfs-hardened.service' \
-      --replace '/usr/bin/ipfs' "$out/bin/ipfs"
```

Comment:
tested via 

```
[nix-shell:~/.cache/nixpkgs-review/pr-259745-1]$ which ipfs
/nix/store/m1cn25jpc5mcf8p7n72rfw2r2arb36rw-kubo-0.23.0/bin/ipfs
```

and 

```
[nix-shell:~/.cache/nixpkgs-review/pr-259745-1]$ ipfs --version
ipfs version 0.23.0
```

Reply:
Well that patch only affects the systemd units, so you didn't actually test that. You could open the unit files in the out path and see that the path to the ipfs binary is replaced properly. Or you could observe that `substituteInPlace` does not complain that it could not find anything.

Reply:
Oh right, brain fart on my part. :o